### PR TITLE
General Theme improvement, with improved Atom and Molecule

### PR DESCRIPTION
**Pull request by andrepeiry**
Remove not implemented elements for code coverage. Expand theme to have a wider range of available colors. Update Atom and Molecule to have better function signature, as well as default values.

- Remove not-yet implemented Atom and Molecule (code coverage).
- Improve Theme by assigning our values instead of default ones.
- Simplify and expand Atom and Molecule signatures by allowing `Modifier` as a argument, as well as adding default values, for the top/bottom bars for example.

Code coverage of this PR
image

Close #39 and #30.

### CODE REVIEW

File: app/src/main/java/com/github/se/cyrcle/ui/theme/Theme.kt [hunk 1]
```diff
@@ -5,22 +5,99 @@ import androidx.compose.material3.MaterialTheme
...
-/* Other default colors to override if/when needed
-background = Color(0xFFFFFBFE),
-surface = Color(0xFFFFFBFE),
-onPrimary = Color.White,
-onSecondary = Color.White,
-onTertiary = Color.White,
-onBackground = Color(0xFF1C1B1F),
-onSurface = Color(0xFF1C1B1F),
-*/
+    lightColorScheme(
```
Comment (matttsss) [inline app/src/main/java/com/github/se/cyrcle/ui/theme/Theme.kt:63]: I think there was a confusion between light and dark mode here

**Review by sean-prz — APPROVED** [review]
This Pull Request was fixed during a meeting and checked with all teams member

**Review by matttsss — APPROVED** [review]
All good, the colors are now fixed, and we have a clear coverage!

**Comment by sean-prz** [comment]
> 

and without removing any code !

**Pull request merged**